org.mwc.cmap.NarrativeViewer/src/org/mwc/cmap/NarrativeViewer/NarrativeViewerModel.java

```diff
@@ -483,7 +528,16 @@ public int getColumnWidth()
 
     public Object getProperty(final NarrativeEntry entry)
     {
-      return myTimeFormatter.format(entry.getDTG());
+      HiResDate dtg = entry.getDTG();

```

Comment:
@IanMayo I just review HiResDate it use **_micros** in hashcode method and it is imutable so user can not change the object so **HiResDate** is imutable so safe to use in WeakhashMap key